test/http_multi_server_test.dart

```diff
@@ -237,6 +237,65 @@ void main() {
       }
     });
   });
+
+  group("HttpMultiServer.bindSecure", () {
+    test("listens on all localhost interfaces for 'localhost'", () async {
+      final server = await HttpMultiServer.bindSecure(
+          "localhost", 0, SecurityContext.defaultContext);
+      server.listen((request) {
+        request.response.write("got request");
+        request.response.close();
+      });
+
+      if (await supportsIPv4) {
+        expect(http.read("https://127.0.0.1:${server.port}/"),
```

Comment:
These fail:

```
HandshakeException: Connection terminated during handshake
```

I think we need something other than `SecurityContext.defaultContext`, or maybe we need to do something different for the `http.read` call... In any case I'm not sure the right way to test this.